python/hail/expr/functions.py

```diff
@@ -855,6 +855,21 @@ def pl_dosage(pl) -> Float64Expression:
     likelihoods with uniform prior. Only defined for bi-allelic variants. The
     `pl` argument must be length 3.
 
+    For a PL array ``[a, b, c]`` let:
+
+    .. math::
+
+        a^\\prime := 10^{a/10} \\\\
```

Comment:
The explosion of backslashes always bothers me. Could you make it a raw string (`r"""..."""`) and delete every other `\`?